src/header/classes.scss

```diff
@@ -1,11 +1,73 @@
 // Copyright (c) Bentley Systems, Incorporated. All rights reserved.
 // See LICENSE.md in the project root for license terms and full copyright notice.
 @import './index';
+@import '../breadcrumbs/index';
 
 .iui-page-header {
   @include iui-page-header;
+  @include iui-page-header-slim;
 }
 
-.iui-header-logo {
-  @include iui-header-logo;
+.iui-page-header-left,
+.iui-page-header-center,
+.iui-page-header-right {
+  @include iui-page-header-zone;
+}
+
+.iui-page-header-right {
+  @include iui-page-header-right;
+}
+
+.iui-page-header-divider {
+  @include iui-page-header-divider;
+}
+
+.iui-header-brand {
+  @include iui-header-brand;
+}
+
+.iui-header-brand-icon {
+  @include iui-header-brand-icon;
+}
+
+.iui-header-brand-label {
+  @include iui-header-brand-label;
+}
+
+.iui-header-breadcrumbs {
+  @include iui-breadcrumbs($breadcrumbs-height: 100%);
+}
+
+.iui-header-breadcrumbs-list {
+  @include iui-breadcrumbs-list;
+}
+
+.iui-page-header-left,
+.iui-header-breadcrumbs,
+.iui-header-breadcrumbs-list {
+  @include iui-page-header-left;
+}
+
+.iui-header-breadcrumb-button {
+  @include iui-header-breadcrumb-button;
+}
+
+.iui-header-breadcrumb-button-split {
+  @include iui-header-breadcrumb-button-split;
+}
+
+.iui-header-breadcrumb-button-container {
```

Comment:
nit: I think this makes more sense as a `-wrapper` than a `-container`.

I like to think of container as something that has multiple instances of something else. e.g. tile container holds multiple tiles. But in this case it is just applied to every `li`.

Reply:
I like that differentiator between wrapper and container, I've made the change.